.github/workflows/test.yml

```diff
@@ -0,0 +1,20 @@
+name: test
+
+on:
+  push:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: [ 14, 16, 18 ]
```

Comment:
Since this lib is only for browser, we do not need to test on multiple Node versions.

Reply:
@leader22 Thank you for your review :)

I fixed this at c446a8c . Since 16 is latest lts, I left this version.